tools/rimage/src/include/rimage/sof/user/manifest.h

```diff
@@ -25,6 +25,7 @@
 /* module type load type */
 #define SOF_MAN_MOD_TYPE_BUILTIN	0
 #define SOF_MAN_MOD_TYPE_MODULE		1
+#define SOF_MAN_MOD_TYPE_LLEXT		2
```

Comment:
maybe explain what LL and EXT stand for? it's not obvious to me. Low Latency, Loadable Library, External, Extension, Exploding,  who knows?

Acronyms are evil.

Reply:
this conversation made me curious, what is llext stands for? ;) @lyakh

Reply:
https://www.bing.com/search?q=zephyr+llext

Reply:
But it's a fair point this file has nothing to do with Zephyr, so it's not so obvious to figure out this is a term coming from Zephyr.